Lib/asyncio/taskgroups.py

```diff
@@ -2,7 +2,7 @@
 # license: PSFL.
 
 
-__all__ = ["TaskGroup"]
+__all__ = ("TaskGroup",)
```

Comment:
I think this change is unnecessary. CPython core devs as a rule prefer diff which little as possible.

Reply:
It is necessary because the others are all tuples and all the `__all__` are summed together in `__init__`. If this is a list, it raises an error.

Reply:
> It is necessary because the others are all tuples and all the `__all__` are summed together in `__init__`. If this is a list, it raises an error.

Oh, you're right. Sorry for the noise